WebDevStudios/nextjs-wordpress-starter
# Feature/next image refactor

Closes N/A

### Link

https://nextjs-wordpress-starter-fwk956x8s-webdevstudios.vercel.app/

### Description

- Smarter support for next/image by checking the domain list in `next.config.js` before trying to use next/image
- Fixes associated Stories
- Fixes PropTypes errors for Search page

### Screenshot

![screenshot](https://dl.dropbox.com/s/n0dix7unjsdh4bu/Screen%20Shot%202021-03-31%20at%2013.56.03.png?dl=0)

### Verification

How will a stakeholder test this?

1. Add and image block with an image from the Media Library. This should use next/image. (e.g., image on https://nextjs-wordpress-starter-fwk956x8s-webdevstudios.vercel.app/)
1. Add an image block with an external image (from Unsplash or something). This should use the HTML image tag. (e.g., image on https://nextjs-wordpress-starter-fwk956x8s-webdevstudios.vercel.app/404-not-found)
1. Verify each display properly
